wrappers/golang/internal/generator/poseidon/templates/poseidon_test.go.tmpl

```diff
@@ -0,0 +1,59 @@
+package tests
+
+import (
+	"testing"
+	
+	core "github.com/ingonyama-zk/icicle/v2/wrappers/golang/core"
+	cr "github.com/ingonyama-zk/icicle/v2/wrappers/golang/cuda_runtime"
+	{{.Field}} "github.com/ingonyama-zk/icicle/v2/wrappers/golang/{{.BaseImportPath}}"
+	poseidon "github.com/ingonyama-zk/icicle/v2/wrappers/golang/{{.BaseImportPath}}/poseidon"
+
+	{{if eq .Field "bls12_381"}}
```

Comment:
I think you can remove this debug now?

Reply:
Its being used to create the result string from limbs to test against.